hrms/src/main/java/kodlamaio/hrms/api/controllers/JobPostingsController.java

```diff
@@ -27,22 +29,22 @@ public JobPostingsController(JobPostingService jobPostingService) {
 	}
 	
 	@PostMapping
-	public void add(@RequestBody JobPosting jobPosting) {
-		jobPostingService.add(jobPosting);		
+	public Result add(@RequestBody JobPosting jobPosting) {
+		return this.jobPostingService.add(jobPosting);		
 	}
 	
 	@DeleteMapping("{id}")
-	public void delete(@PathVariable int id) {
-		jobPostingService.delete(id);
+	public Result delete(@PathVariable int id) {
+		return this.jobPostingService.delete(id);
 	}
 	
 	@GetMapping("{id}")
-	public Optional<JobPosting> get(@PathVariable int id){
+	public DataResult<Optional<JobPosting>> get(@PathVariable int id){
```

Comment:
1 tane @PathVariable yeter yukarıda zaten tanımlamışsın
```suggestion
	public DataResult<Optional<JobPosting>> get( int id){
```